pkgs/top-level/all-packages.nix

```diff
@@ -1202,9 +1202,9 @@ with pkgs;
 
   # The full-featured Git.
   gitFull = git.override {
-    svnSupport = true;
+    svnSupport = stdenv.buildPlatform == stdenv.hostPlatform;
     guiSupport = true;
-    sendEmailSupport = true;
+    sendEmailSupport = stdenv.buildPlatform == stdenv.hostPlatform;
```

Comment:
BTW, I think the condition should be `stdenv.buildPlatform.canExecute stdenv.hostPlatform`, to handle some edge cases (i686 and x86_64?). See https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/cross-compilation.chapter.md#my-packages-testsuite-needs-to-run-host-platform-code-cross-testsuite-runs-host-code

Reply:
Ah, right, I guess that condition shouldn't be used blindly. My bad.